### DISCUSSION TITLE
😴 Nothing interesting yet 😴

### DISCUSSION THREAD

**Discussion by LivingWithHippos** [LivingWithHippos/unchained-android#169] [Feedback]
### 🏷️ New features:

- [x] new authentication engine under the hood
- [x] new telemetry backend (debug versions only)

### 🔎  Enhancements:

- [x] 3 new search plugins
- [x] updated translations

### 🦟 Bug fixes:

- [x] it should crash less

#### Notes:

- as usual there are two APKs, a release and a debug one (the bigger one with "debug" in the name), the debug version will allow me to fix bugs more easily but if your app does not crash you don' t need to use it
- both releases on GitHub have all the search plugins pre-installed
- the new authentication engine will need more testing by you, the ~~beta tester~~ end user! It should crash less and manage the authentication flow better. You may need to re-authenticate on the first run.
- it will be uploaded on Google Play a little later

<hr /><em>This discussion was created from the release <a href='https://github.com/LivingWithHippos/unchained-android/releases/tag/4.20.1-beta'>😴 Nothing interesting yet 😴</a>.</em>

**Comment by boognish-rising** [comment]
Thanks for the additional plugins!

**Reply by LivingWithHippos** [reply]
You're welcome! I only create new ones on request because I find 99.9% of what I need on 133tx

**Reply by boognish-rising** [reply]
Yeah I hear that, I have some obscure-ish taste in music, or at the very least stuff that's often not easy to track down, so the more options the better but etree meets most of those needs TBH

**Reply by sanjosanjo** [reply]
I can't get the 1337x.unchanged file to load into the Unchanged app.  How did you get it to install?

**Reply by LivingWithHippos** [reply]
> I can't get the 1337x.unchanged file to load into the Unchanged app. How did you get it to install?

You have multiple options:

1. Use your phone browser to go on the plugin page: https://github.com/LivingWithHippos/unchained-android/blob/master/extra_assets/plugins/1337.unchained and then share the page with unchained
2. You can download it and share it to unchained with a file manager
3. If you can't install it you can always use the apk from github instead of the one from google play. They have all the plugins already installed

If it works it should show plugin installed.
